.github/workflows/emulator-sample-install.yml

```
@@ -0,0 +1,48 @@
+name: Workflow Sample Install macOS
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
+
+env:
+  CMAKE_VERSION: "3.10.2.4988404"
+  NDK_VERSION: "21.0.6113669"
```

```suggestion
  NDK_VERSION: "21.2.6472646"
```

github actions has this installed

---

we have a fixed version on the build file
https://github.com/getsentry/sentry-android/blob/master/sentry-android-ndk/build.gradle.kts#L41

21.2.x is not available on LGTM GH App. yet.